src/main/java/at/hannibal2/skyhanni/config/features/crimsonisle/ReputationHelperConfig.java

```diff
@@ -23,6 +23,11 @@ public class ReputationHelperConfig {
     @ConfigEditorBoolean
     public boolean useHotkey = false;
 
+    @Expose
+    @ConfigOption(name = "Hide Completed", desc = "Hides tasks after they've been completed.")
+    @ConfigEditorBoolean
+    public boolean hideComplete = true;
```

Comment:
Make this boolean a Property<Boolean> so we can listen to its value change and update the render.